editoast/src/client/mod.rs

```diff
@@ -37,6 +38,12 @@ pub struct GenerateArgs {
     pub force: bool,
 }
 
+#[derive(Args, Debug)]
+#[clap(about, long_about = "Clear generated data")]
+pub struct ClearArgs {
+    pub infra_ids: Vec<u32>,
```

Comment:
Add field documentation. You could add documentation to the `GenerateArgs` struct too.